packages/timeline-state-resolver-types/src/generated/httpSend.ts

```diff
@@ -6,42 +6,10 @@
  */
 
 export interface HTTPSendOptions {
-	/**
-	 * Whether a makeReady should be treated as a reset of the device. It should be assumed clean, with the queue discarded, and state reapplied from empty
-	 */
-	makeReadyDoesReset?: boolean
 	/**
 	 * Minimum time in ms before a command is resent, set to <= 0 or undefined to disable
 	 */
 	resendTime?: number
-	makeReadyCommands?: HTTPSendCommandContent[]
-}
-export interface HTTPSendCommandContent {
```

Comment:
This change might conflict slightly with https://github.com/nrkno/sofie-timeline-state-resolver/pull/269/files, it would be good to make sure all changes from releaase50 are in the release51 branch and rebase this on top to make sure it does all work.